src/parse.rs

```diff
@@ -10,9 +10,10 @@ use quote::quote;
 use syn::{parse::Result as ParseResult, spanned::Spanned as _, Error as SynError};
 
 const ATTR_NAME: &str = "property";
-
+const SKIP: &str = "skip";
 const GET_TYPE_OPTIONS: (&str, Option<&[&str]>) = ("type", Some(&["ref", "copy", "clone"]));
 const SET_TYPE_OPTIONS: (&str, Option<&[&str]>) = ("type", Some(&["ref", "own"]));
```

Comment:
I think maybe you can just add an option here:

```suggestion
const SET_TYPE_OPTIONS: (&str, Option<&[&str]>) = ("type", Some(&["ref", "own", "none"]));
```
(Maybe the name `type` is not proper, I will rename it in future as a break change)

And add codes in the follow parts:
- Add a `None_` item:
  https://github.com/yangby-cryptape/rust-property/blob/10439969773b5ce09184455060a5522e5fc3c89e/src/parse.rs#L43-L46
- Deserialize `none` to `SetTypeConf::None_`:
  https://github.com/yangby-cryptape/rust-property/blob/10439969773b5ce09184455060a5522e5fc3c89e/src/parse.rs#L190-L195
- Generate codes:
  https://github.com/yangby-cryptape/rust-property/blob/10439969773b5ce09184455060a5522e5fc3c89e/src/lib.rs#L159-L198